src/screens/DashboardScreen.tsx

```diff
@@ -180,11 +180,7 @@ export default function DashboardScreen(): JSX.Element {
                 </Button>
             </div>
             <div>
-                {watchQueue ? (
-                    <ShowCarousel data={watchQueue} />
-                ) : (
-                    <ShowCarouselPlaceholder count={5} />
-                )}
+                {watchQueue ? <ShowCarousel data={watchQueue} /> : <ShowCarouselLoader count={5} />}
```

Comment:
@Thenlie 
Is this syntax preferred over:
```
{watchQueue ? (
                    <ShowCarousel data={watchQueue} />
                ) : (
                    <ShowCarouselPlaceholder count={5} />
                )}
```
Just curious about your thoughts. I've always used the prior syntax.

Reply:
I think eslint made this change when the component name got a bit shorter